std/digest/sha.d

```diff
@@ -216,11 +213,20 @@ struct SHA(uint hashBlockSize, uint digestSize)
         version(USE_SSSE3)
         {
             import core.cpuid : ssse3;
-            import std.internal.digest.sha_SSSE3 : transformSSSE3;
+            import std.internal.digest.sha_SSSE3 : sse3_constants=constants, transformSSSE3;
 
             static void transform(uint[5]* state, const(ubyte[64])* block) pure nothrow @nogc
             {
-               return ssse3 ? transformSSSE3(state, block) : transformX86(state, block);
+                if (ssse3)
+                {
+                    version (D_InlineAsm_X86_64)
+                        // constants as extra argument for PIC, see Bugzilla 9378
+                        transformSSSE3(state, block, &sse3_constants);
+                    else
+                        transformSSSE3(state, block);
+                }
+                else
+                    transformX86(state, block);
```

Comment:
It's run as part of the 32-bit shared library test suite (b/c that is still PIC incompatible).